msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -311,11 +303,8 @@ public static void create(@NonNull final Context context,
      *                             and <a href="https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki">MSAL Github Wiki</a>
      *                             </p>
      * @return An instance of IPublicClientApplication.
-     * @throws IllegalStateException if this function is invoked on the main thread.
```

Comment:
Should we keep this? (or explicitly call out that this function can be invoked on a bg thread only)

Reply:
here and below